psh/test.yaml

```diff
@@ -6,6 +6,7 @@ test:
     tests:
         - name: gibber
           harness: test-gibber.py
+          reboot: True
```

Comment:
Please add a comment why we want to reboot there. Sth like `# psh tests need more resources, especially on smaller syspage targets, we don't want to have a test app from the previous test in the campaign loaded`